test/c3/appframework/appframework_test.go

```diff
@@ -211,4 +212,83 @@ var _ = Describe("c3appfw test", func() {
 			testenv.VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV1, false, true)
 		})
 	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
```

Comment:
Cannot add this test to C3 without IDXC. SVA does not apply to this test as its only deploying SH. I would recommend creating a C3 Cluster in whole or move this test out.